connect-examples/node-connect-postgres/postgres.js

```diff
@@ -0,0 +1,75 @@
+var pg = require('pg')
+var through = require('through2').obj
+var xtend = require('xtend')
+
+module.exports = pgStream
+
+function pgStream (app, config, ready) {
+  var pool = new pg.Pool(config)
+
+  var UPDATE =  {
+    text: 'INSERT INTO apps(app, stream_offset) VALUES ($1, $2) ON CONFLICT (app) DO UPDATE SET stream_offset=$2 RETURNING (stream_offset)',
```

Comment:
Stores the offset, overwriting whatever is there. If it's the first go round we'll just write 0.